Runtime/Core/UnityVectorExtensions.cs

```diff
@@ -107,6 +107,11 @@ public static Vector3 ProjectOntoPlane(this Vector3 vector, Vector3 planeNormal)
             {
                 // The lines are parallel (or close enough to it).
                 intersection = Vector2.positiveInfinity;
+                if ((p1 - p3).sqrMagnitude < 0.01f || (p1 - p4).sqrMagnitude < 0.01f ||
+                    (p2 - p3).sqrMagnitude < 0.01f || (p2 - p4).sqrMagnitude < 0.01f)
```

Comment:
Is this intentional?  Square of epsilon is smaller than epsilon, not bigger.

Reply:
Epsilon was too small